### ISSUE TITLE
Add different color schemes

### ISSUE THREAD

**Issue by grthor** [jcryptool/core#177]
It would be a nice gimmik, if a user could choose a color scheme for the JCT, something like a light or dark mode. 

With e4 you can add different color schemes to an rcp: https://www.vogella.com/tutorials/Eclipse4CSS/article.html

Goal of this issue is, to allow the user to select a color scheme in the preferences, like in the eclipse IDE. The color scheme selection is shown in the following screenshot.

![image](https://user-images.githubusercontent.com/20046726/76408246-f67a4180-638c-11ea-90a9-a5ba7cf3d02b.png)

**Comment by grthor** [comment]
Since creating different color schemes in a pure Eclipse 4 RCP is easier than in an e4 RCP with compatibility layer, I would recommend to address this issue only after the e4 migration.

**Comment by simlei** [comment]
Since this is out of scope for e3/e4 compatibility layer which we are still on for the foreseeable future, this gets closed until we migrate.
